docs/rgb_decomposition.md

```diff
@@ -0,0 +1,53 @@
+RGB Decomposition
+=================
+
+RGB products generated with HyP3 use both co-pol (VV or HH) and cross-pol (VH or HV) backscatter values to generate a color image from SAR data. This approach facilitates visual interpretation by decomposing the signals into surface scattering with some volume scattering (red band), volume scattering (green band), and surface scattering with very low volume scattering (blue band). 
+
+The dual-pol SAR dataset first undergoes radiometric terrain correction (RTC) to adjust for the distortions present in the images due to the side-looking geometry of SAR data acquisition. The RTC images are output in amplitude scale, and the RGB values are calculated using a combination of the VV and VH (or HH and HV) pixel values. The output RGB GeoTIFF is scaled so that the values for each band range from 1 to 255.
```

Comment:
No need to switch back to VV and VH (or HH and HV) here -- just stick with co-pol and cross-pol. 

Also, probably should mention 0 = no data

Reply:
Good idea on the no-data. The document actually had the scaling from 0-255, which I knew we'd changed.